internal/release/main.go

```diff
@@ -0,0 +1,218 @@
+// package main gets the next semver version for the given package using git history
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"sort"
+	"strings"
+
+	"github.com/Masterminds/semver"
+	git "github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/go-git/go-git/v5/plumbing/storer"
+	"golang.org/x/mod/modfile"
+)
+
+func getProjectPath() (string, error) {
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		return "", fmt.Errorf("cannot get current file path")
+	}
+	return filepath.Join(filepath.Dir(filename), "..", ".."), nil
+}
+
+func printNewVersion(module string) error {
+	// Ensure the path is consistent with the go mod
+	baseProjectPath, err := getProjectPath()
+	if err != nil {
+		return fmt.Errorf("cannot get current file path")
+	}
+	modPath := filepath.Join(baseProjectPath, module, "go.mod")
+	bytes, err := os.ReadFile(modPath)
+	if err != nil {
+		return err
+	}
+	modFile, err := modfile.Parse(modPath, bytes, nil)
+	if err != nil {
+		return err
+	}
+	if !strings.HasSuffix(modFile.Module.Mod.Path, module) {
+		return fmt.Errorf("module is not named consistently with path %s", module)
+	}
+
+	filteredTags, err := getModuleTags(module)
+	if err != nil {
+		return err
+	}
+
+	vs := make([]*semver.Version, len(filteredTags))
```

Comment:
```suggestion
	versions := make([]*semver.Version, len(filteredTags))
```

a little clearer, when I see `vs` I think `VirtualService`